client/state/test/index.js

```diff
@@ -1,14 +1,40 @@
-require( 'lib/react-test-env-setup' )();
 /**
  * External dependencies
  */
-import { expect } from 'chai';
+import Chai, { expect } from 'chai';
 import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import nock from 'nock';
 
-const createReduxStore = require( 'state' ).createReduxStore;
+/**
+ * Internal dependencies
+ */
+import domEnvSetup from 'lib/react-test-env-setup';
 
 describe( 'state', () => {
+	before( () => {
+		Chai.use( sinonChai );
```

Comment:
I really like that we are doing setup and tear down on the top level, especially I'm happy about disabling all network requests :+1: